admin-cli/README-cli.md

```diff
@@ -0,0 +1,52 @@
+# admin-cli
+
+This is a command line tool for catalog admins to examine the contents of the catalog
+database.
+
+To use it:
+
+1. Build:
+```shell
+./gradlew :admin-cli:bootJar
+```
+
+2. Start the database proxy. `ENV` is the database environment to use. Note that you currently
+must use port `5432`, which may conflict with a locally running postgres. If the port is
+in use, `db_connect.sh` will report an error and exit.
+
+```shell
+ENV=dev PORT=5432 ./scripts/db_connect.sh
+```
+
+In the proxy output, look for the database name, username and password.
+
+3. Run the command:
+
+```shell
+DATABASE_NAME=catalog DATABASE_USER=catalog DATABASE_USER_PASSWORD=<password> \
```

Comment:
I don't know where are passwords are stored offhand, but if they are on vault, could we include a vault path here? (Maybe even in a ${} so that this is easy to copy and paste?) I know I just always love when I can just copy and paste a command from a readme and it works offhand.

Reply:
The database name, username and password are printed out as a result of running the `db_connect.sh` script, so if you follow these steps you'd have that information already. I can add a note about that here. I could also modify the db_connect script to print out the env var setting line to cut & paste, as it currently does for the `jdbc` url and `psql` command.

I was thinking that it wouldn't be _too_ hard to write a wrapper script that smooths over this part of the process. It could run the db_connect script and capture the database information. It could find the library .jar and call `java -jar` on it, forwarding any command line arguments. Maybe that would be a future enhancement though.

Reply:
Ah, yeah. A wrapper script would probably be good, but for the time being I would add that the information is part of the output of db_connect to the readme